nistats/tests/test_first_level_model.py

```diff
@@ -197,6 +197,24 @@ def test_high_level_glm_null_contrasts():
     np.testing.assert_almost_equal(get_data(z1), get_data(z2))
 
 
+def test_high_level_glm_different_design_matrices():
+    # test that one can estimate a contrast when design matrices are different
+    # new API
```

Comment:
~# new API~
\>1.5yr old API

We don't have to add this comment anymore.
I don't even know what it means, there's no explanation for it and it's sprinkled at many places.